src/ui/src/js/controllers/admin_user_index.js

```diff
@@ -41,35 +44,46 @@ export function adminUserIndexController($scope, $uibModal, UserService) {
     );
   };
 
-  function loadUsers() {
+  $scope.doSync = function() {
+    $uibModal.open({
+      controller: 'SyncUsersController',
+      size: 'md',
+      template: syncUsersTemplate,
+    }).result.then(() => {
+      loadUsers();
+    }, () => {
+      loadUsers();
+    });
+  };
+
+  const loadUsers = function() {
     $scope.response = undefined;
-    $scope.data = {};
+    $scope.users = [];
 
     UserService.getUsers().then($scope.successCallback, $scope.failureCallback);
-  }
+  };
+
+  const setUserFullySynced = function() {
+    $scope.users.forEach((user) => {
+      user.fullySynced = true;
+
+      Object.values(user.sync_status).forEach((synced) => {
+        // If we get here at all, then there is sync data and we'll want
+        // to render it, so set that here. If we never get here, that means
+        // there are no remote gardens, so showing the sync status would be
+        // meaningless.
+        $scope.displaySyncStatus = true;
+
+        if (!synced) {
+          user.fullySynced = false;
+        }
+      });
+    });
+  };
 
   $scope.$on('userChange', () => {
     loadUsers();
   });
 
   loadUsers();
 }
-
-newUserController.$inject = ['$scope', '$uibModalInstance'];
```

Comment:
I moved this controller into its own file.  I needed to add another controller for the user index page and felt like it belonged in its own file, so I just moved this one too to keep it consistent.